vanessaspear/passport-client
# Landing page now displays background video, company name, and slogan

## Description

The landing page now reflects the app's purpose and shares the company's name and slogan with the user.

## Changes 
- Added a background video 
- Add some navbar formatting 
- Add text with company name and slogan

## Steps to test 
```
git fetch origin vs-landing-video
git checkout vs-landing-video
```
- Verify the landing page has the video playing and that the navbar text has changed to blue
- Switch to a small viewport.  Verify the navbar logo has disappeared
